docs/user/ppl/cmd/information_schema.rst

```diff
@@ -0,0 +1,98 @@
+=======================================
+information_schema metadata queries
+=======================================
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Description
+============
+| Use ``information_schema`` in source command to query tables information under a catalog.
+In the current state, ``information_schema`` only support metadata of tables.
+This schema will be extended for views, columns and other metadata info in future.
+
+
+Syntax
+============
+source = <catalog>.information_schema.tables;
+
+catalog is optional. If catalog is not provided, it defaults to default catalog ``@opensearch``.
+
+Example 1: Fetch tables in prometheus catalog.
+==============================================
+
+The examples fetches tables in the prometheus catalog.
+
+PPL query for fetching PROMETHEUS TABLES with where clause::
+
+    os> source = my_prometheus.information_schema.tables | where TABLE_NAME='prometheus_http_requests_total'
+    fetched rows / total rows = 1/1
+    +-----------------+----------------+--------------------------------+--------------+--------+---------------------------+
+    | TABLE_CATALOG   | TABLE_SCHEMA   | TABLE_NAME                     | TABLE_TYPE   | UNIT   | REMARKS                   |
+    |-----------------+----------------+--------------------------------+--------------+--------+---------------------------|
+    | my_prometheus      | default        | prometheus_http_requests_total | counter      |        | Counter of HTTP requests. |
```

Comment:
The result is not well formatted. https://github.com/vamsi-amazon/sql/blob/describe-table-prometheus/docs/user/ppl/cmd/information_schema.rst

Reply:
Strange.... do doc tests works without proper formatting.

will correct.